test/fixtures/manifest-file

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I don't think we need to add a new fixture file. We can use any of the existing ones.